UnityProject/Assets/Scripts/Items/Others/Candle.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+using Systems.Clothing;
+
+namespace Items.Others
+{
+	[RequireComponent(typeof(ItemLightControl))]
+	public class Candle : NetworkBehaviour, ICheckedInteractable<InventoryApply>, IServerDespawn
+	{
+		private ItemLightControl lightControl;
+
+		[SerializeField]
+		private SpriteHandler spriteHandler = default;
+
+		[SyncVar] public int LifeSpan = 120; //10 minutes
+		[SyncVar] public int DecayStage = 0;
+		[SyncVar] private bool IsOn = false;
```

Comment:
You don't need to sync var this stuff, the sprite changes will be automatically sent over to clients